OpenRobertaServer/staticResources/js/helper/msg.js

```diff
@@ -11,11 +11,17 @@ define([ 'exports', 'log', 'jquery', 'blocks-msg' ], function(exports, LOG, $, B
             $('#messageConfirm').attr('lkey', lkey);
             $('#messageConfirm').html(value + Blockly.Msg.POPUP_CONFIRM_CONTINUE);
             $("#show-message-confirm").modal("show");
-        } else {
-            $('#message').attr('lkey', lkey);
-            $('#message').html(value);
-            $("#show-message").modal("show");
-        }
+        } else {    
```

Comment:
Instead of this `if` you could pass an optional parameter "opt_switch" or something to avoid direct comparisons to messages.